language/documentation/book/src/bool.md

```diff
@@ -16,7 +16,7 @@ Literals for `bool` are either `true` or `false`.
 | Syntax   | Description | Equivalent Expression |
 | -------- | ----------- | --------------------- |
 | `&&` | short-circuiting logical and | `p && q` is equivalent to `if (p) q else false` |
-| <code>&#124;</code> | short-circuiting logical or |`p <code>&#124;</code> q` is equivalent to `if (p) true else q` |
+| `||` | short-circuiting logical or |`p || q` is equivalent to `if (p) true else q` |
```

Comment:
This one still doesn't look quite right to me--maybe markdown needs an escape?
![image](https://user-images.githubusercontent.com/3513142/121604927-49173300-ca00-11eb-81b3-2832f5e1d964.png)

Reply:
Good point. I am unsure how to render this locally, the markdown viewer I was using seemed happy with it.
If you could let me know what's the best way to get the same rendering I'll make sure it renders fine!

Reply:
I have the same problem with local viewers... I usually use this under "files changed" as the source of truth:

![image](https://user-images.githubusercontent.com/3513142/121614685-10815480-ca14-11eb-8175-8226e9027ea4.png)